src/operation.rs

```diff
@@ -48,7 +48,7 @@ impl<'a> PrintJob<'a> {
 
 impl<'a> IppOperation for PrintJob<'a> {
     fn to_ipp_request(&mut self, uri: &str) -> IppRequestResponse {
-        let mut retval = IppRequestResponse::new(PRINT_JOB, uri);
+        let mut retval = IppRequestResponse::new(Operation::PrintJob as u16, uri);
```

Comment:
If we go all the way to enums I think it's better to change the IppRequestResponse constructor method to accept the enum rather than the u16 (also in the diffs below).